### PR TITLE
permutation ("move")

### PR DESCRIPTION
**Pull request by melauob**
test

### CODE REVIEW

File: src/parser.jl [hunk 1]
```diff
@@ -21,16 +21,18 @@ const RATIO_FILE_NAME = "ratios.txt"
 const VEHICLES_FILE_NAME = "vehicles.txt"
 
 # An Instance structure that is used to format data as we want.
-struct Instances
```
Comment (Jonathan-Fontaine44) [inline src/parser.jl:24]: I think you should rename your `INSTANCES` to `Instances`, a full capital word is used for constants.

File: src/parser.jl [hunk 2]
```diff
@@ -21,16 +21,18 @@ const RATIO_FILE_NAME = "ratios.txt"
...
     # objectif function
     HPRC_rank::Int
     LPRC_rank::Int # If there is no LPRC, LPRC_rank = -1
     PCB_rank::Int
     # paint limitation
     nb_paint_limitation::Int
     # ratio constraint
-    HPRC::Array{Float64, 1}
-    LPRC::Array{Float64, 1}
+    HPRC_p::Array{Int, 1}
```
Comment (Jonathan-Fontaine44) [inline src/parser.jl:32]: Those changed have already been proposed in #15, but it seems ok to me.

File: src/parser.jl [hunk 3]
```diff
@@ -42,7 +44,7 @@ struct Instances
 end
 
 # This function is used to read data of an instance from all files, and agregate into an Instance structure.
-function parser(instance_name::String, instance_type::String, path_folder::String="../data/Instances_")
+function parser(instance_name::String, instance_type::String, path_folder::String="/home/lamraoui/Projets/momh-roadef2005/data/Instances_")
```
Comment (Jonathan-Fontaine44) [inline src/parser.jl:47]: You have to change the path_folder default value, It should be "../data/Instances_" according to what has been proposed in #3, and according to the structure of files defined.

**Review by Jonathan-Fontaine44 — COMMENTED** [review]
You should delete the file "Untitled.ipynb", that is not a julia file.

**Comment by Jonathan-Fontaine44** [comment]
And i don't see where is your implementation of "move" as declared in the Pull Request title.

**Comment by remi-garcia** [comment]
Don't merge this until #15 is.

I deleted the `.ipynb`

**Comment by melauob** [comment]
C'est pour la première fois que j'utilise github , du coup j'ai pas su
comment la push correctement

Le mer. 30 oct. 2019 à 22:42, Rémi Garcia <notifications@github.com> a
écrit :

**Comment by remi-garcia** [comment]
No problem, it's a duplicate of minor changes of #15. I'll close this, now you know how to make a PR. If you need anything else don't hesitate.

Pas de problème, il s'agit d'un doublon de changements présents dans #15. Je ferme cette PR, maintenant tu sais faire une PR. Si tu as besoin de quoi que soit n'hésite pas.

**Pull request closed without merging**